tools/otci/otci/command_handlers.py

```diff
@@ -89,6 +99,8 @@ def __init__(self, otcli: OtCliHandler, is_spinel_cli=False):
         self.__otcli_reader.setDaemon(True)
         self.__otcli_reader.start()
 
+        self.__line_read_callback = None
```

Comment:
@simonlingoogle PTAL at latest commit. not entirely sure what you mean by adding as a class member, but I think this will resolve it?

Reply:
@gabekassel Your latest commit works. 

> not entirely sure what you mean by adding as a class member

Instead of initializing `self.__line_read_callback = None` in `__init__`, we can also add `__line_read_callback = None` as a class member even before `__init__`. In such case, all object has `self.__line_read_callback` being `None` automatically, and is more efficient.